core/save-git.c

```diff
@@ -1273,6 +1277,14 @@ int do_git_save(git_repository *repo, const char *branch, const char *remote, bo
 	return 0;
 }
 
+int save_dives_git(const char *loc, const char *branch, const char *user, bool is_remote, bool is_cloud)
```

Comment:
so we now have save_dives_git() and git_save_dives()
that seems confusing
definitely needs more commenting

Reply:
Yes, that's an unfortunate choice of names. But note that these functions are combined to a single one in the next commit. So this pretty much is an artifact.